src/components/execution/PythonLogo.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Tooltip from '@material-ui/core/Tooltip';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import { withTranslation } from 'react-i18next';
+
+const styles = (theme) => ({
+  pythonLogo: {
+    display: 'inside-block',
+    width: '22px',
+  },
+  valid: {
+    fill: theme.palette.primary.main,
+  },
+  invalid: {
+    fill: 'gray',
+  },
+});
+
+const PythonLogo = ({ t, pythonVersion, classes }) => {
+  if (!pythonVersion) return null;
+
+  const { valid, version } = pythonVersion;
+
+  return (
+    <Tooltip
+      title={`Python (${
+        version ? `${t('version')} ${version}` : t('not installed')
```

Comment:
This may also need to be reconciled/made consistent with the `return null` above. If it is not installed, I expect we want to return `null`, right? In which case the condition after `:` should be something other than 'not installed'

Reply:
Ok, now I see what's happening. Makes sense.